potodo/logging.py

```diff
@@ -2,6 +2,9 @@
 
 
 def setup_logging(logging_level: int) -> None:
+    """
+    Setup logging.
```

Comment:
This is pretty obvious, maybe something like this would've been more interesting:

```suggestion
    Setup logging.
    _
    :param logging_level (int): The standard logging level to be set for the logger
```

Reply:
This is the example I give to students to show how *not* do write a docstring, along with things like:

```python
if user.is_authenticated:  # if user is authenticated
```